Resources/doc/index.rst

```diff
@@ -251,8 +251,9 @@ Here is an example on how to inject the service container into your migrations:
         services:
             Doctrine\Migrations\Version\DbalMigrationFactory: ~
             App\Migrations\Factory\MigrationFactoryDecorator:
-                decorates: Doctrine\Migrations\Version\DbalMigrationFactory
-                arguments: ['@App\Migrations\Factory\MigrationFactoryDecorator.inner', '@service_container']
```

Comment:
this code was **not** subject to the deprecation you mention, as it explicitly injects `@service_container` and does not rely on autowiring `Symfony\Component\DependencyInjection\ContainerInterface`

Reply:
Okay, I had not set `@service_container` explicitly but this should be probably fixed with https://github.com/symfony/symfony/pull/38999